docs/1.getting-started/5.third-parties.md

```diff
@@ -0,0 +1,157 @@
+
+![Chrome User Experience Report chart](/assets/docs/getting-started/third-parties/chart.png)
+<sub><sub>
+source: Chrome User Experience Report,
+date: October 2023,
+devices: phone
+</sub></sub>
+
+While the chart's correlation does not automatically indicate causation, lab experiments and data from the [Web Almanac](https://almanac.httparchive.org/en/2022/third-parties) provide further evidence that third-party resources significantly affect page performance.
+
+## Optimizing Third Party Performance
+
+In general, it can be difficult to determine the most optimal way to load different third-party resources. Nuxt provides a number of utilities that include various approaches to improve both the developer and user experience of loading third parties.
+
+::callout
+Please refer to [third-party wrappers](#third-party-wrappers) for available pre-configured solutions.
+::
+
+### useScript
+
+The `useScript` composable enables you to load third-party scripts with SSR support and a proxied API. It works out of the box, requiring just an ID and a URL that points to the script resource.
+
+```vue
+<script setup lang="ts">
+useScript({
+  id: 'some_id',
+  src: 'https://www.example.com/script.js'
+})
+</script>
+```
+
+Optional triggers and asset strategies can be used to allow for more fine-grained control over how, and when, the script should be loaded.
+
+```vue
+<script setup lang="ts">
+useScript({
+  id: 'some_id',
+  src: 'https://www.example.com/script.js',
+  trigger: new Promise((resolve) => {
+    setTimeout(() => { resolve() }, 1000)
+  })
+})
+</script>
+```
+
+> For more details about triggers, asset strategies and other supported options, please refer to [link].
+
+### useStyles
+
+The `useStyles` composable enables you to load third-party stylesheets consistently.
+
+```vue
+<script setup lang="ts">
+useStyles(['https://www.example.com/styles.css'])
+</script>
+```
+
+You can use optional asset strategies to inline styles directly in the `head` section of your `document`.
+
+```vue
+<script setup lang="ts">
+useStyles(
+  ['https://www.example.com/styles.css'],
+  { assetStrategy: 'inline' }
+)
+</script>
+```
+
+> For more details, please refer to [link].
+
+### Third-Party Wrappers
+
+A collection of utilities that are already configured to load and initialize popular third parties in the most performant way. These include:
+
+- Composables to fetch scripts such as `useGoogleAnalytics` and `useGoogleTagManager`
+- Components to load embeds such as `<GoogleMaps>` and `<YoutubeEmbed>`
+
+#### Google Analytics
+
+The `useGoogleAnalytics` composable allows you to integrate Google Analytics in your Nuxt application.
+
+```vue
+<script setup lang="ts">
+useGoogleAnalytics({
+  id: 'GA-123456789-1'
+})
+</script>
+```
+
+Read more in [docs > api > composables > use-gogle-analytics](../3.api/2.composables/use-google-analytics.md).
```

Comment:
```suggestion
Read more in [docs > api > composables > use-google-analytics](../3.api/2.composables/use-google-analytics.md).
```

Reply:
Looking at the other docs, looks there's already a component utility for "Read More" callouts:

```suggestion
:read-more{to="/docs/api/composables/use-google-analytics"}
```